pkg/operator/helpers/helpers.go

```diff
@@ -55,12 +55,16 @@ const (
 	// DefaultAddonNamespace is the default namespace for agent addon
 	DefaultAddonNamespace = "open-cluster-management-agent-addon"
 
+	LabelPrefix = "open-cluster-management.io"
+
 	// HubLabelKey is used to filter resources in informers
-	HubLabelKey = "createdByClusterManager"
+	HubLabelKey = LabelPrefix + "/created-by-clusterManager"
```

Comment:
Camel case and - mixed looks weird, created-by-clustermanager or created-by-cluster-manager are both acceptable to me.